src/main/java/org/tikv/common/exception/KeyException.java

```diff
@@ -26,6 +29,7 @@ public KeyException(String errMsg) {
   }
```

Comment:
Better to change this into `this(null)`.

Reply:
> The other ctor explicitly called super already.
This is why we should call the other ctor which has called super, why "so this is not right"?

Reply:
this(null) in turns calls
```
public KeyException(Kvrpcpb.KeyError keyErr) {
    super("Key exception occurred");
    this.keyErr = keyErr;
  }
```
which initialize error message to "Key exception occurred".
If really want to optimize this, there should be a ctor like:
```
public KeyException(String errMsg, Kvrpcpb.KeyError keyErr) {
    super(errMsg);
    this.keyErr = keyErr;
}
```
And other ctor in turns calls this one.

Reply:
Woo! I didn't notice that. Thanks for pointing it out